sympy/geometry/polygon.py

```diff
@@ -212,19 +210,22 @@ def _isright(a, b, c):
         ========
 
         >>> from sympy import Point, Polygon
-        >>> a, b, c = [Point(i) for i in [(0, 0), (1, 1), (1, 0)]]
+        >>> a, b, c, d = [Point(i) for i in [(0, 0), (1, 1), (1, 0), (2, 2)]]
         >>> Polygon._isright(a, b, c)
-        True
+        1
```

Comment:
`True` as an output is better.

Reply:
It was intended to make clear collinear and the other two conditions